src/ontology/uberon-edit.obo

```diff
@@ -224531,6 +224531,148 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600108
+name: proximal ureter
+def: "A segment of an ureter that extends from the renal pelvis to the common iliac artery bifurcation (to form the internal and external iliac arteries)." [PMID:30422575]
+synonym: "abdominal ureter" EXACT [PMID:30422575]
+synonym: "upper ureter" EXACT [PMID:30422575]
+xref: FMA:15882
+is_a: UBERON:0006555 ! excretory tube
+relationship: continuous_with UBERON:8600109 ! distal ureter
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+relationship: dc-contributor https://orcid.org/0000-0001-7655-4833 ! Ellen Quardokus
+relationship: dc-contributor https://orcid.org/0000-0002-4341-3265
+relationship: dc-contributor https://orcid.org/0000-0003-2804-127X ! Sanjay Jain
+relationship: part_of UBERON:0000056 ! ureter
+property_value: dcterms-date "2024-08-28T13:51:59Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600109
+name: distal ureter
+def: "A segment of ureter that extends from the common iliac artery bifurcation to the bladder. From there, the ureter curves back and follows the lateral pelvic wall inferiorly to connect with the posterolateral bladder wall." [PMID:30422575]
+synonym: "lower ureter" EXACT [PMID:30422575]
+synonym: "pelvic ureter" EXACT [PMID:30422575]
+xref: FMA:15883
+is_a: UBERON:0006555 ! excretory tube
+relationship: continuous_with UBERON:8600108 ! proximal ureter
+relationship: continuous_with UBERON:8600110 ! intramural ureter
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+relationship: dc-contributor https://orcid.org/0000-0001-7655-4833 ! Ellen Quardokus
+relationship: dc-contributor https://orcid.org/0000-0002-4341-3265
+relationship: dc-contributor https://orcid.org/0000-0003-2804-127X ! Sanjay Jain
+relationship: part_of UBERON:0000056 ! ureter
+property_value: dcterms-date "2024-08-28T14:15:03Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600110
+name: intramural ureter
+def: "The terminal segment of the ureter that is continuous with the distal ureter and lies where the ureter enters the bladder. The intramural ureter is aperistaltic, serving as a reflux-preventing mechanism, preventing the backflow of urine from the bladder into the ureters." [PMID:14142154, PMID:30422575]
+synonym: "intravesical ureter" EXACT [doi:10.53347/rID-23183]
```

Comment:
Found this reference as well PMID: [33085409](https://pubmed.ncbi.nlm.nih.gov/33085409)